keyboards/royal_kludge/rk68_rgb/config.h

```diff
@@ -26,3 +26,4 @@
 /* Configure the effects:                                              */
 #define RGB_MATRIX_TYPING_HEATMAP_DECREASE_DELAY_MS 50
 
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

Comment:
this should probably be in `keymaps/via/config.h`, since dynamic keymaps get triggered with a VIA build and not a default one. Slightly cleaner approach. Other than that, PR is good to go

Reply:
Done!